plugin.video.crackle/resources/lib/globals.py

```diff
@@ -0,0 +1,198 @@
+import sys, os
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+import urllib, urllib2
+import json
+import base64, hmac, hashlib
+from datetime import datetime
```

Comment:
There are reports about issues with `datetime` module on some platforms, so I'd recommend to use POSIX `time` module for working with time values.